build_runner/lib/src/server/graph_viz_main.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:convert';
```

Comment:
why does the file name mention graph viz?

Reply:
I just wanted it to be unique, its the entrypoint for the asset graph visualizer